readme.md

```diff
@@ -49,6 +49,27 @@ Level and trials:
 6. 120 trials
 7. 240 trials
 
+##### disableBitDepthReduction
```

Comment:
Not a fan of double negative. I would rather drop the "disable" prefix and make the default "true" instead.